scripts/openstack-quickstart-demosetup

```diff
@@ -589,8 +589,6 @@ if [ "x$with_tempest" = "xyes" ]; then
     crudini --set $c DEFAULT service_plugins "neutron_lbaas.services.loadbalancer.plugin.LoadBalancerPluginv2, neutron.services.l3_router.l3_router_plugin.L3RouterPlugin, neutron.services.vpn.plugin.VPNDriverPlugin, neutron.services.metering.metering_plugin.MeteringPlugin, firewall_v2, neutron.services.qos.qos_plugin.QoSPlugin, trunk"
```

Comment:
I think you need to remove it here as well.

Reply:
Done